pl.js

```diff
@@ -0,0 +1,688 @@
+windows.loadout.color.hair Kolor włosów
+windows.loadout.dye Barwnik
+windows.loadout.pet Zwierzak
+windows.loadout.stat Statystyki przy zabiciu
+windows.loadout.flag Flaga kraju
+windows.mods.drop upuść tu plik modyfikacji lub naciśnij
+windows.mods.paste Wklej URL modyfikacji
+windows.mods.load Załaduj modyfikacje
+windows.mods.none Nie znaleziono modyfikacji
+windows.mods.header Menedżer modyfikacji
+windows.mods.guide Poradnik tworzenia modyfikacji
+windows.mods.list Lista modyfikacji
+windows.mods.load.header Załaduj modyfikacje
+windows.mods.publish Opublikuj modyfikacje
+windows.mods.viewer Widz
+windows.account.header Konto
+windows.account.recover Aby przywrócic konto, skontaktuj się z <a>krunker@yendis.ch</a>
+windows.class.header Wybierz klasę
+windows.controls.header Zmień ustawienia kontroli
+windows.controls.forward Naprzód
+windows.controls.backward Do tyłu
+windows.controls.left Lewo
+windows.controls.right Prawo
+windows.controls.reload Przeładuj
+windows.controls.aim Celuj
+windows.controls.shoot Strzelaj
+windows.controls.inspect Pokaż broń
+windows.controls.spray Sprej
+windows.controls.jump Skakanie
+windows.controls.crouch Przyklęknięcie
+windows.controls.prim Główna broń
+windows.controls.swap Druga broń
+windows.controls.melee Broń biała
+windows.controls.equip Wyposażenie
+windows.controls.chat Czat
+windows.controls.voice Głos
+windows.controls.list Lista graczy
+windows.controls.board Tabela wyników
+windows.controls.interact Działanie
+windows.controls.interactSec Drugie działanie
+windows.controls.confirm Potwierdź działanie
+windows.controls.drop Upuść
+windows.controls.wepVis Przełącz widoczność broni
+windows.controls.streak0 Seria 1
+windows.controls.streak1 Seria 2
+windows.controls.streak2 Seria 3
+windows.controls.streak3 Seria 4
+windows.controls.streak4 Seria 5
+windows.controls.premium0 Noclip
+windows.controls.premium1 Tryb boga
+windows.controls.premium2 Zabicie
+windows.controls.premium3 Porażenie
+windows.controls.reset Resetuj pozycję (Parkour i wyścigi)
+windows.controls.pretend Ukryj grę
+windows.host.header Hostuj niestandardową grę
+windows.host.maps Ustaw serwer
+windows.host.cmaps Gra społeczności
+windows.host.raw Surowe dane gry
+windows.host.modes Wybierz tryby
+windows.host.classes Wybierz klasy
+windows.host.settings Ustawienia serwera
+windows.host.preset Zapisane ustawienia
+windows.host.preset.load Ładowanie zapisanego ustawienia...
+windows.host.preset.name Nazwa zapisanego ustawienia
+windows.host.start Rozpocznij grę
+windows.host.rehost Rozpocznij grę (Przybierz graczy)
+windows.spray.header Wybierz sprej
+windows.spray.default Darmowy sprej
+windows.spray.spin Sprej
+windows.maps.header Niestandardowe gry
+windows.maps.none Nie znaleziono gier!
+windows.maps.name Nazwa gry
+windows.maps.editor Edytor gier
+windows.maps.publish Opublikuj grę
+windows.maps.community Społeczność Dev
+windows.publish.map.header Opublikuj lub zaaktualizuj grę
+windows.publish.map.thumbnail Bez miniaturki
```

Comment:
For the sake of consistency, this should be "Brak miniaturki"

Reply:
.